tests/test_datayaml.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import pytz
+import yaml
+
+from datetime import datetime
+
+
+KEYS_CONTAINING_DATES = ['last_updated', 'expires', 'sent']
+
+
+def get_alerts_data(path):
+    with open(path) as file:
+        alerts_data = yaml.load(file, Loader=yaml.FullLoader)
```

Comment:
Minor: we can just do the following, which would make it easier to mock the loading (related to my comment about the test 👇).

```
return yaml.load(file, Loader=yaml.FullLoader)
```

The `with` statement is only really necessary for long running processes, and even then, Python will usually clean up the file handle at some point.